vNext/extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -446,6 +463,15 @@ export class AjaxMonitor implements ITelemetryPlugin, IDependenciesPlugin, IInst
                 if (appId) {
                     xhr.setRequestHeader(RequestHeaders.requestContextHeader, RequestHeaders.requestContextAppIdFormat + appId);
                 }
+                if (this._config.distributedTracingMode === DistributedTracingModes.AI_AND_W3C || this._config.distributedTracingMode === DistributedTracingModes.W3C) {
+                    let traceparent;
+                    if (this._context && this._context.telemetryTrace && this._context.telemetryTrace.traceID) {
+                        traceparent = new Traceparent(this._context.telemetryTrace.traceID, ajaxData.spanId);
+                    } else {
+                        traceparent = new Traceparent(ajaxData.spanId);
```

Comment:
I think its unclear here why this `spanId` is 32 chars but other if, its 16. Should probably be cleaned up so code is more clear